server/services/topics.js

```diff
@@ -0,0 +1,23 @@
+const caseInsensitive = new Intl.Collator('en', { caseFirst: 'lower' });
+
+class TopicsService {
+  constructor(hubMenuRepository, cmsApi) {
+    this.hubMenuRepository = hubMenuRepository;
+    this.cmsApi = cmsApi;
+  }
+
+  async getTopics(prisonId) {
+    const [tags, primary] = await Promise.all([
+      this.hubMenuRepository.tagsMenu(prisonId),
+      this.cmsApi.primaryMenu(prisonId),
+    ]);
```

Comment:
now making requests in parallel and sorting by whole string rather than first letter.